### ISSUE TITLE
Anneal the learning rate during training

### ISSUE THREAD

**Issue by dgoekmen** [RSMI-NE/RSMI-NE#10]
Should be possible to do it the same way the Gumbel softmax temperature is annealed.
